src/ig/IGExporter.ts

```diff
@@ -29,6 +30,17 @@ import { ConfigurationMenuItem } from '../fshtypes';
 import { logger, Type, getFilesRecursive } from '../utils';
 import { FHIRDefinitions } from '../fhirdefs';
 import { Configuration } from '../fshtypes';
+import { parseCodeLexeme } from '../import';
+
+function isR4(fhirVersion: string[]) {
+  let containsR4Version = false;
+  fhirVersion.forEach(v => {
+    if (/(^4\.0\.)|(^(4\.1\.|4\.3.\d+-))|(^4\.3.\d+$)/.test(v)) {
```

Comment:
This seems overly complex.  Wouldn't it be sufficient just to check if it starts w/ 4.0., 4.1., or 4.3.?
```suggestion
    if (/^4\.[013]\./.test(v)) {
```

Reply:
Whoops yeah it would! I had just quickly copied the regular expressions we use elsewhere and never looked back. Oops.